components/hab/src/command/bldr/job/start.rs

```diff
@@ -29,6 +29,8 @@ pub fn start(ui: &mut UI,
                 ui.warn(rdep.to_string())?;
             }
 
+            ui.warn("(Note: dependencies from private origins may not be shown)")
```

Comment:
I believe that the phrase "may not be shown" is going to be too vague here and leave the user wondering why they wouldn't have been shown and in what circumstance(s).

Perhaps something like the following expresses the idea more explicitly:
`Note: dependencies from private origins are omitted for non-members.`

Reply:
Small but important: the `ui.warn` line is missing a semi-colon at the end 😄

Reply:
Sounds reasonable, thanks!